templates/home/_fsf_flutter.html

```diff
@@ -16,28 +16,27 @@ <h4>Why are snaps good for Flutter projects?</h4>
   </div>
 
   <div class="col-6">
-    <h4>Here's how <a href="https://snapcraft.io/flutter-gallery">flutter-gallery</a> defines snapcraft.yaml:</h4>
+    <h4>Here's how <a href="https://snapcraft.io/super-cool-app">super-cool-app</a> defines snapcraft.yaml:</h4>
```

Comment:
@kenvandine are we going to post that super cool app in the store?

Reply:
It's published in the store now